shared/src/common/types/schema/schema.ts

```diff
@@ -0,0 +1 @@
+export type { Schema } from 'joi';
```

Comment:
```suggestion
export type { Schema as ValidationSchema } from 'joi';
```
rename file to validation-schema